main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def get_players(filename: str) -> dict:
+    with open(filename) as file:
+        return json.load(file)
+
+
 def main() -> None:
-    pass
+    players = get_players("players.json")
+
+    for nickname, player_info in players.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_info["race"]["name"],
+            description=player_info["race"]["description"]
+        )
+        for skill in player_info["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race
+            )
+
+        guild = None
```

Comment:
to make it more clear you can do like this `guild = player_info["guild"]`
And then check `if guild` in the next condition